### ISSUE TITLE
Allow font to be changed via accessibility options

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#961]
Would it be useful to add a new option to the accessibility dialog that allows users to change the project font? For example, there are fonts that are designed to increase readability for people with dyslexia (https://gumroad.com/l/OpenDyslexic)

Any thought?

**Comment by ronm123** [comment]
@FayCross yes I think it would. Might be best to include it as an option under author control where the author could decide to disable that option but have it enabled by default. Fonts can vary dramatically in size for the same font size though so we might need to carefully test for layout across all page types and themes if that is the case.

**Comment by FayCross** [comment]
I'm going to close this. I don't think it's particularly important and unlikely to be actioned.
